tests/FSharp.Test.Utilities/TestFramework.fs

```diff
@@ -141,7 +142,7 @@ module Commands =
     let copy workDir source dest =
         log "copy /y %s %s" source dest
         File.Copy( source |> getfullpath workDir, dest |> getfullpath workDir, true)
-        CmdResult.Success
+        CmdResult.Success ""
```

Comment:
Below on line 187 there's an `#if FSC_IN_PROCESS` which has a not updated `CmdResult.Success`. Not sure if that's ever used anymore.

Reply:
Ah yes this is never in use.